### ISSUE TITLE
html output not showing on gradio platform

### ISSUE THREAD

**Issue by mohitin** [gradio-app/gradio#5843]
### Describe the bug

I have a html file that contains exported output of ipysigma, something that looks like \link{https://medialab.github.io/ipysigma/demo.html}, I want to integrate this html file in the gradio platform by reading it in a file but I'm not able to do that

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

#   file_path="/test2.html"

# #   with open(file_path, 'r') as file:
#     #   html_content = file.read()
# #   html_content = html_content.replace("'", "\"")
# #   print(html_content)
# #   return html_content
#   iframe_code = f'<iframe src="{html_content}" width="100%" height="600px" frameborder="0" allowfullscreen></iframe>'
#   return iframe_code

### Logs

```shell
No output
```

### System Info

```shell
Yes
```

### Severity

Blocking usage of gradio

**Comment by abidlabs** [comment]
Hi @mohitin your reproduction is incomplete and hard to follow. Can you please provide a minimal example that is exhibiting this issue?

**Comment by abidlabs** [comment]
Closing for lack of follow up. If you are able to provide us with more details, we can reopen.
